apps/client/src/routes/protectedRoutes.tsx

```diff
@@ -1,16 +1,14 @@
-import LoginForm from '../login/LoginForm';
+import Login from '../pages/Login/Login';
 import ErrorPage from '../pages/ErrorPage';
 import routes from './routes';
 
-// Temporarily disable protected routes.
-// TO-DO: put routes behind authentication.
 const protectedRoutes = [
   {
     path: '/',
-    element: <LoginForm />,
+    element: <Login />,
     errorElement: <ErrorPage />,
+    children: routes,
   },
-  ...routes,
 ];
```

Comment:
Non-blocking (change out of scope of PR): Checkout https://github.com/remix-run/react-router/tree/dev/examples/auth. I think there's a better pattern we can follow.